openshift-hack/build-rpms.sh

```diff
@@ -31,7 +31,7 @@ fi
 os::build::rpm::get_nvra_vars
 
 OS_RPM_SPECFILE="$( find "${OS_ROOT}" -name '*.spec' )"
-OS_RPM_NAME="$( rpmspec -q --qf '%{name}\n' "${OS_RPM_SPECFILE}" | head -1 )"
+OS_RPM_NAME="$( (rpmspec -q --qf '%{name}\n' "${OS_RPM_SPECFILE}"; e="$?"; [ "$e" -eq 141 ] && exit 0; exit "$e") | head -1 )"
```

Comment:
Let's change this to capturing the output from `rpmspec` in a variable. And then `head -1 <<<"${var}"`